XML Project.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.10.35122.118
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "XML Project", "XML Project.csproj", "{DDEB76E9-310C-4943-9A88-139347340A76}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sips&Bites", "Sips&Bites.csproj", "{DDEB76E9-310C-4943-9A88-139347340A76}"
```

Comment:
Project name should be relevant with the application name for better readability